docs/en/setup/backend/otel-collector-config.yaml

```diff
@@ -0,0 +1,184 @@
+apiVersion: v1
```

Comment:
Once you have a YAML, you need ASF header.

Reply:
Added